language/de-DE/de-DE.com_users.ini

```diff
@@ -49,114 +49,118 @@ COM_USERS_LOGIN_REMEMBER_ME="Angemeldet bleiben"
 COM_USERS_REMIND_REQUEST_ERROR="Fehler bei der Anfrage nach einem vergessen Passwort."
 COM_USERS_REMIND_REQUEST_FAILED="Erinnerung fehlgeschlagen: %s"
-COM_USERS_REMIND_REQUEST_SUCCESS="Erinnerung verschickt! Bitte den Posteingang des E-Mail-Kontos prüfen."
+COM_USERS_REMIND_REQUEST_SUCCESS="Erinnerung verschickt. Bitte den Posteingang des E-Mail-Kontos prüfen."
+COM_USERS_REMIND_SUPERADMIN_BODY="Hallo,\n\nes wurde eine Anfrage zum Zurücksetzen des Benutzerpassworts für %s gestellt. Aus Sicherheitsgründen kann ein Super Benutzer jedoch keine Kennworterinnerung anfordern. Bitte an einen anderen Super Benutzer wenden oder eine alternative Methode verwenden, wie in der Dokumentation beschrieben: %s"
 COM_USERS_REMIND_SUPERADMIN_ERROR="Als Super Benutzer kann für das eigene Passwort keine Erinnerung angefordert werden. Bitte einen anderen Super Benutzer kontaktieren oder eine andere Methode wählen."
 COM_USERS_RESET="Passwort zurücksetzen"
-COM_USERS_RESET_COMPLETE_ERROR="Zurücksetzen des Passworts <strong>fehlgeschlagen</strong>!"
+COM_USERS_RESET_COMPLETE_ERROR="Zurücksetzen des Passworts <strong>fehlgeschlagen</strong>."
 COM_USERS_RESET_COMPLETE_FAILED="Das Zurücksetzen des Passworts ist fehlgeschlagen: %s"
 COM_USERS_RESET_COMPLETE_LABEL="Um den Vorgang abzuschließen wird noch ein neues Passwort benötigt."
-COM_USERS_RESET_COMPLETE_SUCCESS="Das Passwort wurde zurückgesetzt! Eine Anmeldung mit dem neuen Passwort ist nun möglich."
-COM_USERS_RESET_CONFIRM_ERROR="Fehler bei der Bestätigung des Passworts!"
-COM_USERS_RESET_CONFIRM_FAILED="Die Bestätigung der Zurücksetzung des Passworts ist fehlgeschlagen, da der Bestätigungscode ungültig ist! %s"
+COM_USERS_RESET_COMPLETE_SUCCESS="Das Passwort wurde zurückgesetzt. Eine Anmeldung mit dem neuen Passwort ist nun möglich."
+COM_USERS_RESET_CONFIRM_ERROR="Fehler bei der Bestätigung des Passworts."
+COM_USERS_RESET_CONFIRM_FAILED="Die Bestätigung der Zurücksetzung des Passworts ist fehlgeschlagen, da der Bestätigungscode ungültig ist. %s"
 COM_USERS_RESET_CONFIRM_LABEL="Eine Nachricht mit weiteren Informationen wurde an die E-Mail-Adresse verschickt. Die E-Mail enthält einen Bestätigungscode, dieser muss kopiert und in das unten stehende Feld eingefügt werden, damit sichergestellt wird, dass nur der Eigentümer des Benutzerkontos diese Änderung durchführen kann."
-COM_USERS_RESET_COMPLETE_TOKENS_MISSING="Die Bestätigung zum Zurücksetzen des Passworts ist fehlgeschlagen, da der Bestätigungscode fehlt!"
-COM_USERS_RESET_REQUEST_ERROR="Fehler bei der Anfrage zum Zurücksetzen des Passworts!"
+COM_USERS_RESET_COMPLETE_TOKENS_MISSING="Die Bestätigung zum Zurücksetzen des Passworts ist fehlgeschlagen, da der Bestätigungscode fehlt."
+COM_USERS_RESET_REQUEST="Wenn die eingegebene E-Mail-Adresse auf dieser Website registriert ist, erhält man in Kürze eine E-Mail mit einem Link zum Zurücksetzen des Passworts für das Benutzerkonto."
```

Comment:
```suggestion
COM_USERS_RESET_REQUEST="Wenn die eingegebene E-Mail-Adresse auf dieser Website registriert ist, erhalten Sie in Kürze eine E-Mail mit einem Link zum Zurücksetzen des Passworts für das verknüpfte Benutzerkonto."
```

Reply:
Please check. I have deliberately tried not to use "Sie".